doc/build.md

```diff
@@ -128,7 +128,7 @@ As before, node 1 will create the blockchain and begin mining. As we'll be runni
 
 First, we run a wallet server to receive rewards on port 15000 (we'll log in debug mode for more information about what's happening)
```

Comment:
I think the language in the documentation should be updated as well, something like `run a wallet service to listen and receive rewards on ...`.